### PR TITLE
All README badges now use results from respective master branches

### LINKED ISSUE #25: Badges should only reflect upstream master
Currently the badges show failed status when other users submit prs that fail

### PR DESCRIPTION
**Pull request by Rabadash8820**
Closes #25. My "proof" that this will work is provided there.

If you're cool with these changes, then of course similar PRs will need to be opened in each of the other Unity Action repos.

### CODE REVIEW

**Comment by Rabadash8820** [comment]
Ugh, I'm assuming all those checks failed cause I didn't have any `UNITY_*` secrets defined in my fork (forgot this would be a different fork than `unity-builder`). Probably simplest for me to add a `UNITY_LICENSE`, since there still seems to be issues (#26) with the Plus licensing model. I won't be able to re-run the checks though.

**Comment by Rabadash8820** [comment]
K yeah, just added a `UNITY_LICENSE` secret for a Personal license to my fork. Maybe I can just find a typo to fix in the README or something, so I can make another commit to trigger another workflow run...

**Comment by Rabadash8820** [comment]
Hm, after adding `UNITY_EMAIL` and `UNITY_PASSWORD` to my fork and triggering another workflow run things are still failing. The fact that these checks are failing shouldn't affect the new badges working, but it is annoying...

Sorry @webbertakken, I know we have conversations going across like 15 Issues/PRs and two repos now...

**Comment by Rabadash8820** [comment]
Ok so actually I think you're fine. According to those same docs (like one sentence later 🤦‍♂): `By default, badges display the status of your default branch (usually master).` As proof of that, check out the badges below. The default `badge.svg` badge and the one with `branch=master` are both the same. As another test, you can see that the badge on your current `master` README is green, even though the last couple runs of your main workflow have failed due to my PR #28.

`Actions status`

Actions status

`Actions status`

Actions status

`Actions status`

Actions status

**Comment by Rabadash8820** [comment]
So why did the badge of the Builder repo turn red after the check for webbertakken/unity-builder#14 failed? I think that was because I opened a PR from the *`master`* branch of my fork, so presumably GitHub Action's badge system can't distinguish `master` in a fork from `master` in the base repo. I bet that if I had opened a PR from a branch with a different name (which I will certainly do in the future), then *your* repo's badge would've been unaffected.

LONG story short, I think you can close this PR once you've soaked all of this in. 😛

**Comment by Rabadash8820** [comment]
As for why all the checks for this PR kept failing, I have a theory. Its related to #27, so I'm still doing some research, but I know this much so far: when GitHub Actions are triggered in response to PRs from a fork, those Actions run [in the *base repo*](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#pull-request-events-for-forked-repositories). So I was kind of at the mercy of whatever secrets you had set in this repo. I would think that you already set up the licensing secrets in this repo correctly, but maybe something wasn't? 🤷‍♂ Maybe the checks for the PRs in the `unity-builder` repo are failing for similar reasons?

**Comment by webbertakken** [comment]
Good effort! Closing this as requested. I guess we just need a single way that works and done :)
